tests/test_modeling_common.py

```diff
@@ -460,6 +464,88 @@ def _create_and_check_torchscript(self, config, inputs_dict):
 
             self.assertTrue(models_equal)
 
+    def test_torch_fx(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        self._create_and_check_torch_fx_tracing(config, inputs_dict)
+
+    def test_torch_fx_output_loss(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        self._create_and_check_torch_fx_tracing(config, inputs_dict, output_loss=True)
```

Comment:
How heavy in terms of memory/processing power are those tests? It makes me think of the Keras tests that we had to eventually disable as it took more than a couple of minutes per model class, which wasn't feasible for CI.

Reply:
From: https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/5bdabdd888af1f000130874a/360197618/609eb688e2ebd83fbb07349a-0-build/artifacts/~/transformers/reports/tests_torch_durations.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210514T180650Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210514%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=2f132f54ff812769cb205d414bf497061226c7c6fd2260ef02711b19e540050c
```
0.71s call     tests/test_modeling_electra.py::ElectraModelTest::test_torch_fx
0.62s call     tests/test_modeling_t5.py::T5ModelTest::test_torch_fx
3.71s call     tests/test_modeling_bert.py::BertModelTest::test_torch_fx
```

Reply:
Yes I would say that's because there are more model classes for BERT than for T5.

Reply:
Ah! the answer was simple then ;) Thank you!